### PR TITLE
Upgrade simapp to nightly-2020-08-04

### LINKED ISSUE #336: Upgrade simapp to nightly-2020-08-04
Depends on https://github.com/CosmWasm/cosmjs/pull/323

I had some issues with our scripts when updating to anyting > nightly-2020-07-29. This needs some investigation.

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #336

### CODE REVIEW

**Comment by webmaster128** [comment]
Can you try a run of `./generate_template.sh`. This was the part that was failing for me.

File: scripts/simapp/setup.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -o errexit -o nounset -o pipefail
+set -o errexit -o nounset
```
Comment (webmaster128) [inline scripts/simapp/setup.sh:2]: Why this change?

File: scripts/simapp/setup.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ simd init --chain-id "$CHAIN_ID" "$MONIKER"
 sed -i "s/\"stake\"/\"$STAKE\"/" "$HOME"/.simapp/config/genesis.json # staking/governance token is hardcoded in config, change this
 
 echo "Setting up validator ..."
-if ! simd keys show validator; then
+if ! simd keys show validator 2> /dev/null; then
```
Comment (webmaster128) [inline scripts/simapp/setup.sh:16]: 👍

File: scripts/simapp/template/.simapp/config/config.toml [hunk 3]
```diff
@@ -263,13 +263,13 @@ version = "v0"
...
-timeout_precommit = "300ms"
-timeout_precommit_delta = "100ms"
-timeout_commit = "1s"
+timeout_propose = "3s"
+timeout_propose_delta = "500ms"
+timeout_prevote = "1s"
+timeout_prevote_delta = "500ms"
+timeout_precommit = "1s"
+timeout_precommit_delta = "500ms"
+timeout_commit = "5s"
```
Comment (webmaster128) [inline scripts/simapp/template/.simapp/config/config.toml:272]: This block should be reverted

File: scripts/simapp/setup.sh (hunk 1 above)
Comment (willclarktech) [reply scripts/simapp/setup.sh:2]:
```
In ./setup.sh line 2:
set -o errexit -o nounset -o pipefail
                             ^------^ SC2039: In POSIX sh, set option pipefail is undefined.

For more information:
  https://www.shellcheck.net/wiki/SC2039 -- In POSIX sh, set option pipefail ...
```

Comment (webmaster128) [reply scripts/simapp/setup.sh:2]: I see. Can we switch from `#!/bin/sh` to `#!/bin/bash` in order to keep the feature?

Comment (willclarktech) [reply scripts/simapp/setup.sh:2]:
```
docker run --rm -it interchainio/simapp:nightly-2020-08-05 sh
~ # bash
sh: bash: not found
```

Comment (webmaster128) [reply scripts/simapp/setup.sh:2]: Ah, the image is based on busybox (low end system for routers and stuff). Alright then, thanks for checking!
